### PR TITLE
[Payments NOX] Integrate incentives logic into frontend

### LINKED ISSUE #51453: [WC Payments NOX] Integrate WooPayments incentives UI
### Description

If there is an active WooPayments incentive, surface it on the Payments settings page.

### Acceptance criteria

* The action incentive UI is present when there is an active incentive.
* The switch incentive UI is present when there is an active incentive. 

### Designs
Design post: pfHfG4-q7-p2
Figma: xO8YdpKfaVoP2d8NvweZ5I-fi-1428_21437

### Dev notes
Reactifying the Payments settings page: pe4Hk0-1ly-p2
Populate react-powered main payments settings screen PR: https://github.com/woocommerce/woocommerce/pull/50825

### Additional context
PRD: 1wMfX8CAjDnOp24tcU-PADB0pOt3cz7mGdyZYeRBxx2Y-gdoc

cc @timmy5685

### PR DESCRIPTION
**Pull request by dmallory42**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes #52453 

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

0. Run `pnpm i && pnpm build`
1. Pull the latest `trunk` on your local server installation and the latest `develop` on your local WooPayments.
2. Make sure there are no activated payment extensions, including WooPayments. (You can disable it in `WP Admin > Plugins`)
3. Checkout this PR's branch. 
4. Apply [this patch](https://gist.githubusercontent.com/vladolaru/9f9c8b18304d507a8b7e95e24962128e/raw/277a0effe4c14d4d5d8be6e94c17d58bcc8dc153/wc-96-wcpay-action-incentive.patch) to meet the Action incentive eligibility criteria.
5. Reload the Payment Settings page.
6. Verify that you can see the Incentive Banner, and that any providers which have an active incentive are highlighted in the interface matching the designs. (see screenshots). The text should match the expected text for an action incentive.
7.  Reset the previous patch that was applied.
8. Enable a different payment gateway (e.g. PayPal).
9. Apply [this patch](https://gist.github.com/dmallory42/0bfbbbea358e26a54224b535df21376a) to meet the Switch incentive eligibility criteria
10. Reload the page, you should see the modal for the switch incentive. You should also see the banner, and the row highlighting. Verify this matches the designs.

### Dismissing an Incentive

1. Apply [this patch](https://gist.githubusercontent.com/vladolaru/9f9c8b18304d507a8b7e95e24962128e/raw/277a0effe4c14d4d5d8be6e94c17d58bcc8dc153/wc-96-wcpay-action-incentive.patch) to meet the Action incentive eligibility criteria.
2. Reload the Payment Settings page.
3. Verify that you can see the Incentive Banner.
4. Click `No thanks` on the Incentive Banner.
5. You should see a request being made to the `/dismiss` endpoint. Verify it returns successfully.
6. The incentive should disappear. Reload the page and the incentive should no longer show. 
7. Reset the previous patch that was applied.

### Accepting an incentive

1. Follow the steps in the troubleshooting section to clear the state of your store so that you can see the extension again.
2. Apply [this patch](https://gist.githubusercontent.com/vladolaru/9f9c8b18304d507a8b7e95e24962128e/raw/277a0effe4c14d4d5d8be6e94c17d58bcc8dc153/wc-96-wcpay-action-incentive.patch) to meet the Action incentive eligibility criteria.
3. Verify you can see the incentive banner. Click the button to accept the incentive.
4. Onboard the account. You should see the incentive details on the `Payments > Overview` page (under account status).

### Troubleshooting

If you don't see the incentives, there are various things you can try to make sure your store is in the right state:

- Make sure you have Marketplace suggestions enabled in `WooCommerce > Settings > WooCommerce.com`
- Delete the `woocommerce_task_list_hidden` option.
- Use the WC Beta Tester plugin to reset WC onboarding and set the timestamp to now. (see screenshot)
- Delete any `wp_option` rows with `incentive` in them (you can query `where option_name LIKE '%incentive%'`

Screenshot 2024-12-09 at 18 36 23

### Screenshots

Incentive banner and row highlighting:

Screenshot 2024-12-05 at 18 41 05

### Designs

xO8YdpKfaVoP2d8NvweZ5I-fi-3106_71471

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

-   [ ] This Pull Request does not require a changelog entry. (Comment required below)

<details>

<summary>Changelog Entry Details</summary>

#### Significance

-   [ ] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [ ] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>

### CODE REVIEW

**Review by anu-rock — COMMENTED** [review]
#### Manual testing

| Scenario  | Result | Screenshot |
| ------------- | ------------- | ------------- |
| Meet the action incentive eligibility criteria  | ✅ The 10% incentive banner was shown as in the screenshot above. | |
| Meet the switch incentive eligibility criteria  | ✅ The 20% incentive banner was shown as in the screenshot above. | |
| Trigger the action incentive banner and click its "no thanks" button  | ❓ The banner was dismissed, but the highlight in WooPayment's row persisted. Is that expected? | image |

___

Suggestion: Change the terms and conditions link in both incentives from https://woocommerce.com/terms-conditions/woopayments-promotion-2023/ to https://woocommerce.com/terms-conditions/woopayments-promotion. It redirects there anyway. "2023" makes it sound outdated.

Question related to switch incentive modal: While the modal is displayed, the banner is also visible in the background. That feels a bit overwhelming. WDYT about showing the banner only when the modal is closed? cc @orcunomattic

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx [hunk 1]
```diff
@@ -14,18 +15,59 @@ import { WC_ASSET_URL } from '~/utils/admin-settings';
...
+	 * Incentive data.
+	 */
+	incentive: PaymentIncentive;
+	/**
+	 * Callback used when an incentive is accepted.
+	 *
+	 * @param id   Plugin ID.
+	 * @param slug Plugin slug.
+	 */
+	setupPlugin: ( id: string, slug: string ) => void;
```
Comment (anu-rock) [inline plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx]: Shouldn't this be called `onAccept`? The current name is too specific, an action performed on accepting.

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx [hunk 2]
```diff
@@ -14,18 +15,59 @@ import { WC_ASSET_URL } from '~/utils/admin-settings';
...
+	/**
+	 * Callback to handle dismiss action.
+	 *
+	 * @param dismissHref Dismiss URL.
+	 * @param context     The context in which the incentive is dismissed.
+	 */
+	onDismiss: ( dismissHref: string, context: string ) => void;
```
Comment (anu-rock) [inline plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx]: Nitpick

```suggestion
	onDismiss: ( dismissUrl: string, context: string ) => void;
```

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx [hunk 3]
```diff
@@ -14,18 +15,59 @@ import { WC_ASSET_URL } from '~/utils/admin-settings';
...
+	 *
+	 * @param id   Plugin ID.
+	 * @param slug Plugin slug.
+	 */
+	setupPlugin: ( id: string, slug: string ) => void;
+	/**
+	 * Callback to handle dismiss action.
+	 *
+	 * @param dismissHref Dismiss URL.
+	 * @param context     The context in which the incentive is dismissed.
```
Comment (anu-rock) [inline plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx]:
```suggestion
	 * @param context     The context in which the incentive is dismissed (eg. CSS class name of the related banner or modal)
```

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx
Comment (anu-rock) [inline plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx:1]: Thanks for making all those unrelated improvements in this component.

File: plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx [hunk 4]
```diff
@@ -18,12 +18,15 @@ import './settings-payments-body.scss';
...
+import { IncentiveBanner } from '~/settings-payments/components/incentive-banner';
+import { IncentiveModal } from '~/settings-payments/components/incentive-modal';
+import { WooPaymentsPostSandboxAccountSetupModal } from '~/settings-payments/components/woo-payments-post-sandbox-account-setup-modal';
 import {
 	getWooPaymentsTestDriveAccountLink,
 	isWooPayments,
 	providersContainWooPaymentsInTestMode,
 } from '~/settings-payments/utils';
-import { WooPaymentsPostSandboxAccountSetupModal } from '~/settings-payments/components/woo-payments-post-sandbox-account-setup-modal';
+import apiFetch from '@wordpress/api-fetch';
```
Comment (anu-rock) [inline plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx]: The linter is showing me these two errors:

```
Expected "External dependencies" to be defined before Internal eslint(@woocommerce/dependency-group)
Expected preceding "External dependencies" comment block eslint(@woocommerce/dependency-group)
```

**Review by anu-rock — COMMENTED** [review]
#### Review checklist

| Parameter  | Result |
| ------------- | ------------- |
| Approach  | ✅ All good |
| Implementation  | 👍  Almost good (left a few comments) |
| Automated tests  | Not applicable |
| Naming  | ✅ All good (minor nitpicks) |
| Typing  | ✅ All good |
| Code comments  | ✅ All good  |
| Changelog  | ❌ Missing. Don't we need an internal one here? |
| Documentation  | Not applicable |
| PR description  | ✅ All good |
| Testing instructions  | ✅ All good |
| Manual testing  | See my last comment |

**Comment by vladolaru** [comment]
> The banner was dismissed, but the highlight in WooPayment's row persisted. Is that expected?

@anu-rock, the incentive is not completely dismissable (at least not right now). The provider row will remain highlighted if an incentive is available.

**Comment by oaratovskyi** [comment]
I have a few comments about the design:
1. Incentive banner styling is different from design - mostly paddings.

See this:
Screenshot 2024-12-10 at 14 59 56

payment providers and incentive banner should match the same padding
image

and also here you can see that padding from the CTA (Save 10%) to bottom is bigger, from top to Limited time offer - too. The incentive image is different on design / impl:
| Design | Implementation |
|--------|-------|
| Screenshot 2024-12-10 at 14 58 27 |Screenshot 2024-12-10 at 14 58 44|


2. There should be tooltip icon 

| Design | Implementation |
|--------|-------|
|  Screenshot 2024-12-10 at 15 00 45 | Screenshot 2024-12-10 at 15 00 54 |

3. The incentive gateway height should be bigger than others (see design):
Screenshot 2024-12-10 at 15 02 23

4. Now there's violet background on gateway if modal if banner is here. It shouldn't be:

| Design | Implementation |
|--------|-------|
|  image  | image |

**Comment by oaratovskyi** [comment]
@dmallory42 another idea - could we move the incentive modal to `components/modals` and re-use the css? or it's too different from others?

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx (hunk 1 above)
Comment (dmallory42) [reply plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx]: Good point - updated in af827c0412.

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx (hunk 2 above)
Comment (dmallory42) [reply plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx]: Thanks! Updated in af827c0412

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx (hunk 3 above)
Comment (dmallory42) [reply plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx]: Updated in 21293460ec, thanks!

File: plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx (hunk 4 above)
Comment (dmallory42) [reply plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx]: Fixed, thanks!

**Comment by dmallory42** [comment]
> @dmallory42 another idea - could we move the incentive modal to components/modals and re-use the css? or it's too different from others?

I think in future, yes - but it will need some refactoring because of differences between the modals, so I'll do this as a follow-up later.

**Comment by dmallory42** [comment]
Hi @oaratovskyi! Thanks for the great eye for detail. I made a few fixes which should address most of the styling/consistency issues, and updated the logic so that we only highlight the row if the banner isn't showing, so it's now like this:

- For an action incentive, we show the banner and don't highlight the row
- For a switch incentive, we show the modal and highlight the row

There are a couple of things that are still outstanding which I will address in smaller, follow-up PRs:

- Showing a tooltip when hovering over the Status Badge
- Correct logic when an incentive is accepted to apply it to the account
- Fixing the image used in the banner so that it cuts off correctly at the top and bottom

Looking forward to your next review!

File: plugins/woocommerce/src/Internal/Admin/Suggestions/Incentives/WooPayments.php [hunk 5]
```diff
@@ -133,11 +133,11 @@ protected function get_incentives( string $country_code ): array {
 			// Store locale, e.g. `en_US`.
 			'locale'       => get_locale(),
 			// WooCommerce store active for duration in seconds.
-			'active_for'   => WCAdminHelper::get_wcadmin_active_for_in_seconds(),
-			'has_orders'   => $this->has_orders(),
+			'active_for'   => 8000000,
```
Comment (anu-rock) [inline plugins/woocommerce/src/Internal/Admin/Suggestions/Incentives/WooPayments.php]: Oops, I think values from the patch in the PR description got checked in.

**Review by anu-rock — APPROVED** [review]
Thanks for the latest revisions. Provisionally approving based on improvements in response to my feedback. I'll let @oaratovskyi give the final word.

File: plugins/woocommerce/src/Internal/Admin/Suggestions/Incentives/WooPayments.php (hunk 5 above)
Comment (oaratovskyi) [reply plugins/woocommerce/src/Internal/Admin/Suggestions/Incentives/WooPayments.php]: Great catch @anu-rock - removed it :)

File: plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways/payment-gateways.tsx [hunk 6]
```diff
@@ -17,7 +17,6 @@ import { decodeEntities } from '@wordpress/html-entities';
 import { CountrySelector } from '~/settings-payments/components/country-selector';
 import { ListPlaceholder } from '~/settings-payments/components/list-placeholder';
 import { PaymentGatewayList } from '~/settings-payments/components/payment-gateway-list';
-import './payment-gateways.scss';
```
Comment (oaratovskyi) [inline plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways/payment-gateways.tsx:20]: I wonder why did you remove it? If it's not needed anymore - let's remove the css too. Or include if it's needed

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx [hunk 7]
```diff
@@ -14,47 +15,77 @@ import { WC_ASSET_URL } from '~/utils/admin-settings';
...
-	const [ isDismissClicked, setIsDismissClicked ] = useState( false );
+	const [ isDismissed, setIsDismissed ] = useState( false );
+	const [ isBusy, setIsBusy ] = useState( false );
+
+	const incentiveContext = 'wc_settings_payments__banner';
 
-	const handleSetup = () => {
+	const handleAccept = () => {
+		setIsBusy( true );
+		onAccept( 'woopayments', 'woocommerce-payments' );
```
Comment (oaratovskyi) [inline plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx:52]: is it up to date? I noticed TODO in the similar code in incentive-modal

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx [hunk 8]
```diff
@@ -14,47 +15,77 @@ import { WC_ASSET_URL } from '~/utils/admin-settings';
...
-							WC_ASSET_URL +
-							'images/settings-payments/incentives-icon.svg'
-						}
-						alt={ __( 'Incentive hero image', 'woocommerce' ) }
-					/>
-				</CardMedia>
+				<img
+					src={
+						WC_ASSET_URL +
+						'images/settings-payments/incentives-icon.svg'
```
Comment (oaratovskyi) [inline plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx]: I'd move it to `images/icons`

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-modal/incentive-modal.tsx [hunk 9]
```diff
@@ -54,7 +91,7 @@ export const IncentiveModal = ( {
 										'images/settings-payments/incentives-icon.svg'
 									}
 									alt={ __(
-										'Incentive hero image',
+										'Incentive icon',
```
Comment (oaratovskyi) [inline plugins/woocommerce/client/admin/client/settings-payments/components/incentive-modal/incentive-modal.tsx]: same here, please move it to `images/icons`

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-modal/incentive-modal.tsx [hunk 10]
```diff
@@ -118,11 +145,16 @@ export const IncentiveModal = ( {
 									disabled={ isBusy }
 									onClick={ () => {
 										setIsBusy( true );
-										onSubmit();
+										// TODO: Temporary for testing, update to use plugin ID and slug.
```
Comment (oaratovskyi) [inline plugins/woocommerce/client/admin/client/settings-payments/components/incentive-modal/incentive-modal.tsx]: could please handle this todo, similar to above?

File: plugins/woocommerce/client/admin/client/settings-payments/components/payment-extension-suggestion-list-item/payment-extension-suggestion-list-item.tsx [hunk 11]
```diff
@@ -31,10 +31,16 @@ export const PaymentExtensionSuggestionListItem = ( {
 	pluginInstalled,
 	...props
 }: PaymentExtensionSuggestionListItemProps ) => {
+	const hasIncentive = !! extension._incentive;
+	const shouldHighlightIncentive =
+		hasIncentive && ! extension._incentive?.promo_id.includes( '-action-' );
```
Comment (oaratovskyi) [inline plugins/woocommerce/client/admin/client/settings-payments/components/payment-extension-suggestion-list-item/payment-extension-suggestion-list-item.tsx:36]: I wonder why only `%-action-%` is supported?

File: plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list-item/payment-gateway-list-item.tsx [hunk 12]
```diff
@@ -25,9 +25,10 @@ export const PaymentGatewayListItem = ( {
 	...props
 }: PaymentGatewayItemProps ) => {
 	const isWcPay = isWooPayments( gateway.id );
+	const hasIncentive = !! gateway._incentive;
+	const shouldHighlightIncentive =
+		hasIncentive && ! gateway._incentive?.promo_id.includes( '-action-' );
```
Comment (oaratovskyi) [inline plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list-item/payment-gateway-list-item.tsx:30]: I wonder why only %-action-% is supported?

File: plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list-item/payment-gateway-list-item.tsx [hunk 13]
```diff
@@ -60,13 +61,10 @@ export const PaymentGatewayListItem = ( {
 				<div className="woocommerce-list__item-text">
 					<span className="woocommerce-list__item-title">
 						{ gateway.title }
-						{ hasIncentive ? (
+						{ hasIncentive && gateway._incentive ? (
```
Comment (oaratovskyi) [inline plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list-item/payment-gateway-list-item.tsx:64]: maybe `hasIncentive` is redundant, not sure

File: plugins/woocommerce/client/admin/client/settings-payments/components/payment-extension-suggestion-list-item/payment-extension-suggestion-list-item.tsx [hunk 14]
```diff
@@ -48,9 +54,15 @@ export const PaymentExtensionSuggestionListItem = ( {
 				<div className="woocommerce-list__item-text">
 					<span className="woocommerce-list__item-title">
 						{ extension.title }{ ' ' }
-						{ isWooPayments( extension.id ) && (
+						{ ! hasIncentive && isWooPayments( extension.id ) && (
 							<StatusBadge status="recommended" />
 						) }
+						{ hasIncentive && extension._incentive && (
```
Comment (oaratovskyi) [inline plugins/woocommerce/client/admin/client/settings-payments/components/payment-extension-suggestion-list-item/payment-extension-suggestion-list-item.tsx:60]: maybe `hasIncentive` is redundant - not sure

File: plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx [hunk 15]
```diff
@@ -93,6 +96,20 @@ export const SettingsPaymentsMain = () => {
 		return select( PLUGINS_STORE_NAME ).getInstalledPlugins();
 	}, [] );
 
+	const dismissIncentive = useCallback(
```
Comment (oaratovskyi) [inline plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx:99]: I'd move it to some util function or even wp-data in future pr 🙂

File: plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list-item/payment-gateway-list-item.tsx (hunk 12 above)
Comment (oaratovskyi) [reply plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list-item/payment-gateway-list-item.tsx:30]: I see, because it's `isActionIncentive`
maybe we could use `! isSwitchIncentive` as we use it in the settings-main

File: plugins/woocommerce/client/admin/client/settings-payments/components/payment-extension-suggestion-list-item/payment-extension-suggestion-list-item.tsx (hunk 11 above)
Comment (oaratovskyi) [reply plugins/woocommerce/client/admin/client/settings-payments/components/payment-extension-suggestion-list-item/payment-extension-suggestion-list-item.tsx:36]: I see, because it's `isActionIncentive`
maybe we could use `! isSwitchIncentive` as we use it in the settings-main

**Review by oaratovskyi — APPROVED** [review]
Small step to go @dmallory42 ! testing instructions check out - pre-approving 🚀 
please address the following code comments that sound good to you!

**Comment by oaratovskyi** [comment]
@dmallory42 created follow up issue https://github.com/woocommerce/woocommerce/issues/53660

File: plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways/payment-gateways.tsx (hunk 6 above)
Comment (dmallory42) [reply plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateways/payment-gateways.tsx:20]: Ooops, I forgot to delete the css file. Done!

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx (hunk 7 above)
Comment (dmallory42) [reply plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx:52]: It's the same TODO for both. The problem we have is that we currently don't detect _which_ incentive is being applied (as in, the gateway it belongs to), because they all belong to WooPayments right now. I can fix it, but will need to do some refactors so I'll address it separately.

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx (hunk 8 above)
Comment (dmallory42) [reply plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx]: I updated the name to `incentives-illustration` - I think it is not really an icon, so the name was a bit misleading!

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx (hunk 7 above)
Comment (oaratovskyi) [reply plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx:52]: okay, in the next PR!

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx (hunk 8 above)
Comment (oaratovskyi) [reply plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.tsx]: TY!

File: plugins/woocommerce/client/admin/client/settings-payments/components/incentive-modal/incentive-modal.tsx (hunk 10 above)
Comment (oaratovskyi) [reply plugins/woocommerce/client/admin/client/settings-payments/components/incentive-modal/incentive-modal.tsx]: same todo for both cases
